packages/aws-cdk/README.md

```diff
@@ -560,6 +561,125 @@ This feature currently has the following limitations:
   bucket). Requires version 12 of the bootstrap stack, for the added
   IAM permissions to the `deploy-role`.
 
+
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ 
+
+CDK migrate is currently experimental and may have breaking changes in the future. 
+
+Generates a CDK application using an existing CloudFormation template in JSON or YAML format. 
+Templates can be provided from either from a local file using --from-path or directly from a 
+deployed CloudFormation stack with --from-stack. The generated CDK application will 
+synthesize a CloudFormation template with identical resource configurations to the provided template. 
+The generated application will be initialized in the current working directory with a single stack where 
+the stack, app, and directory will all be named using the provided `--stack-name`. It will also
+be within a generated subdirectory in your current working directory unless `--output-path` is specified.
+If a directory already exists with the same name as `--stack-name`, it will be replaced with the new application.
+All CDK supported languages are supported, language choice can be specified with `--language`.
+
+#### Generate a typescript application from a local template.json file
+
+```console
+$ # template.json is a valid cloudformation template in the local directory
+$ cdk migrate --stack-name MyAwesomeApplication --language typescript --from-path MyTemplate.json
+```
+
+This command will generate a new directory named `MyAwesomeApplication` within your current working directory, and  
+then initialize a new CDK application within that directory which has the same resource configuration
+as the provided template.json
+
+This results in a CDK application with the following structure, where the lib directory contains a stack definition
+with the same resource configuration as the provided template.json.
+
+```console
+├── README.md
+├── bin
+│   └── my_awesome_application.ts
+├── cdk.json
+├── jest.config.js
+├── lib
+│   └── my_awesome_application-stack.ts
+├── package.json
+├── tsconfig.json
+```
+
+#### Generate a python application from a deployed stack
+
+If you already had a CloudFormation stack deployed in your account and would like to migrate to using CDK instead of CloudFormation.
+You can use the `--from-stack` option to generate the application. In this case the `--stack-name` must match the name of the deployed stack. 
```

Comment:
Super minor, but I think this phrasing of the first sentence here is a little bit too focused on going away from CFN. And it's not strictly accurate. After migrating, the stack will still be managed by CloudFormation, it will just be done via a CDK application. Also, this is not a complete sentence.
```suggestion
If you already have a CloudFormation stack deployed in your account and would like to manage it with CDK, you can use the `--from-stack` option to generate the application. In this case the `--stack-name` must match the name of the deployed stack. 
```